sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveMetastoreCatalog.scala

```diff
@@ -531,18 +531,29 @@ private[hive] class HiveMetastoreCatalog(sparkSession: SparkSession) extends Log
 /**
  * An override of the standard HDFS listing based catalog, that overrides the partition spec with
  * the information from the metastore.
+ * @param tableBasePath The default base path of the Hive metastore table
+ * @param partitionSpec The partition specifications from Hive metastore
  */
 private[hive] class MetaStoreFileCatalog(
```

Comment:
I am inclined to rename this class to `MetastorePartitionedTableFileCatalog` as this is only used for partitioned tables, and therefore does not need to worry about what files to return when the tables is not partitioned.